README.md

```diff
@@ -43,3 +43,21 @@ By participating in this project, you agree to abide by our [code of conduct](./
 - event naming convention: hyphenated lower case.
 	- we don't need to namespace it to individual items because it's such a small list of events. just keep it simple.
 
+## Pubsub events
+
+Events that flow through the global event broker (`app.events`, invoked as
+`this.events` by individual modules):
+
+#### `printable-key`
+
+When the user types a printable key in the urlbar, the complete string in
+the urlbar is fired as the body of the event.
+
+```
+{
+  query: 'the mar'
+}
```

Comment:
It seems like something we'd want to log for metrics, so we could determine e.g. how many times f => facebook.